trikot-viewmodels/swift-extensions/UIButtonExtensions.swift

```diff
@@ -156,7 +156,8 @@ extension UIButton {
         return completedAttributedString
     }
 
-    public func positionSubviews(_ alignment: Alignment?) {
+    @objc
```

Comment:
Why is this required?

Reply:
I needed it to fix this error:

> Non-@objc instance method 'positionSubviews' is declared in extension of 'UIButton' and cannot be overridden

Reply:
Maybe a more elegant way of achieving the same result would have been to conditionally check for a specific protocol conformance instead of overriding a function from an extension.